graphql/schema.graphql

```diff
@@ -88,6 +88,7 @@ type Mutation {
   ): Int!
   createPublicKey(publicKeyInput: PublicKeyInput!): [PublicKey!]!
   spawnHost(spawnHostInput: SpawnHostInput): Host!
+  updateSpawnHostStatus(hostId: String!, action: String!): Host!
```

Comment:
it would be great to use an enum instead of a string for `action` because then we can get input validation and also see list of supported inputs.